src/openms/source/ANALYSIS/MAPMATCHING/QTClusterFinder.cpp

```diff
@@ -99,6 +100,132 @@ namespace OpenMS
     // update parameters (dummy)
     setParameters_(1, 1);
 
+    if(use_IDs_)
+    {
+      std::unordered_map<String, std::vector<double>> ided_feat_rts;
+      for (const auto& map : input_maps)
+      {
+        for (const auto& feat : map)
+        {
+          const auto& pepIDs = feat.getPeptideIdentifications();
+          if (!pepIDs.empty())
+          {
+            //TODO sort (lose const) or assume first is best?
+            if (!pepIDs[0].getHits().empty())
+            {
+              const auto it_inserted = ided_feat_rts.emplace(pepIDs[0].getHits()[0].getSequence().toString() + "/" + feat.getCharge(), std::vector<double>{feat.getRT()});
```

Comment:
add const String key = pepIDs[0].getHits()[0].getSequence().toString() + "/" + feat.getCharge(); for better readability

Reply:
is {feat.getRT()} sufficient?

Reply:
You mean use the whole feature to calculate a note malized distance with FeatureDistance based on mz/RT/Int?
Yes I thought about that too. Do you think it is worth it?
By the way, if we include th experimental design we can have different weights for intensity (across replicates vs across conditions).

Reply:
here I just meant writing {rt} instead of vector<double>{rt}. but both ways is fine imho

Reply:
ah no. Template types are not deduced for initializer_lists